django_webpack/templatetags/webpack.py

```diff
@@ -0,0 +1,12 @@
+from django_webpack.compiler import webpack
+from django_webpack import settings
+from django import template
+import os
+
+register = template.Library()
+
+
+@register.simple_tag
+def webpack_bundle(config_path=settings.CONFIG_PATH):
+    bundle = webpack(os.path.join(settings.BUNDLE_ROOT, config_path))
```

Comment:
I wrote a similar templatetag, but it doesn't seem to need to concatenate BUNDLE_ROOT and the config_path.